map.js

```diff
@@ -1,13 +1,11 @@
 //TODO: Figure out Random Gate tracking?
-    //seperate counter for found random gates?
+    //seperate counter for found random gates, count to 40 and stop drawing remaining gates.
 
-//some decorations from the guide that might be useful.
-    //TODO: Add pink circle for fixed gates 
+//some decorations from the guide that might be useful. //or We could just insert these things into the name of the gates.
```

Comment:
Thinking about this more, we should probably put this info instead on the json oblivion gate entries.

Reply:
yep. it's a Hard Problem and, like fame, will probably take me way too long to implement.